src/pages/permissions/forms/IdpGroupForm.tsx

```diff
@@ -5,9 +5,12 @@ import type { FormikProps } from "formik/dist/types";
 export interface IdpGroupFormValues {
   name: string;
 }
+export interface TLSIdentityFormValues {
+  name: string;
+}
```

Comment:
The type should also include the groups.
```suggestion
export interface TLSIdentityFormValues {
  name: string;
  groups: string[];
}
```

Reply:
Should it? The way I understood it, the formvalues referred to the first formik form that only contained the name input in it.

The logic of adding groups was not changed, and the IDPGroupFormValues did not include a string[] interface for the groups. I believe the two areas are being addressed separately. Am I misunderstanding something here?

Reply:
I see that this suggestion is based off of a later change/suggestion that you have made. Noted!